internal/service/vpc/resource_gcp_vpc_peering_connection.go

```diff
@@ -241,5 +247,25 @@ func copyGCPVPCPeeringConnectionPropertiesFromAPIResponseToTerraform(
 		return diag.FromErr(err)
 	}
 
+	var toProjectId string
+	var toVPCNetwork string
+
+	if len(*peeringConnection.StateInfo) > 0 {
+		if v, ok := (*peeringConnection.StateInfo)["to_project_id"]; ok {
+			toProjectId = v.(string)
+		}
+		if v, ok := (*peeringConnection.StateInfo)["to_vpc_network"]; ok {
+			toVPCNetwork = v.(string)
+		}
+
+		if toProjectId != "" && toVPCNetwork != "" {
+			if err := d.Set("self_link",
+				fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s",
```

Comment:
please make `https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s` const

Reply:
If I do that ID is no longer highlighting  it nicely :disappointed: 
![image](https://user-images.githubusercontent.com/4929004/204796635-517985a3-e813-4c3a-8ca0-153e5f314029.png)
So I suggest lets leave it as it is, WDYT?

Reply:
perhaps we could make the `https://www.googleapis.com/compute/v1` part const then